otherlibs/win32unix/unixsupport.h

```diff
@@ -59,7 +59,9 @@ extern int win_CRT_fd_of_filedescr(value handle);
 
 extern void win32_maperr(DWORD errcode);
 extern value unix_error_of_code (int errcode);
-extern void unix_error (int errcode, char * cmdname, value arg);
+CAMLnoreturn_start
```

Comment:
Why is that change needed? From my quick searches in your top comment, I couldn't find a corresponding warning from the compiler.